assets/stylesheets/host.css

```diff
@@ -9,4 +10,229 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.5);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.5s;
+  background-color: white;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 15px;
+  width: 1000px;
+  height: 550px;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.modal-text {
+  width: 950px;
```

Comment:
I don't think the width needs to be specified at all here.  The browser will fill the whole space, on its own.

Reply:
Good call - yeah I think you're right about this. Done & fixed!